django_graph_api/tests/urls.py

```diff
@@ -1,21 +1,11 @@
-"""test_project URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/1.11/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  url(r'^$', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.conf.urls import url, include
-    2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
-"""
-from django.conf.urls import url
+from django.conf.urls import include, url
 from django.contrib import admin
 
+from django_graph_api.views import GraphQLView
+from .starwars.schema import schema
+
+
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
+    url(r'^admin/', include(admin.site.urls)),
+    url(r'^graphql$', GraphQLView.as_view(schema=schema)),
```

Comment:
In the documentation, we should make a note about needing `ensure_csrf_cookie` if CSRF checks are enabled.

Reply:
I've added ensure_csrf_cookie to the graphql view... they would also need it for any of their own views using it too, though. I've added a note to #13.